base/sparse/sparsematrix.jl

```diff
@@ -2305,13 +2305,14 @@ function getindex(A::SparseMatrixCSC{Tv}, I::AbstractArray) where Tv
     colptrA = A.colptr
     rowvalA = A.rowval
     nzvalA = A.nzval
+    Ti = eltype(colptrA)
```

Comment:
Could perhaps be cleaner to change the function signature to:
```julia
function getindex(A::SparseMatrixCSC{Tv,Ti}, I::AbstractArray) where {Tv,Ti}
    # ...
```

Reply:
Agreed. Made the change.